### PR TITLE
Updated property descriptions

### PR DESCRIPTION
**Pull request by Blackbaud-DeniseCisneros**
Addresses the property updates called out in https://github.com/blackbaud/skyux2-docs/issues/879.

The https://github.com/blackbaud/skyux2-demos/pull/50 PR addresses the UI text feedback.

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-update-error-properties](https://developer.blackbaud.com/skyux?_sp.spa=pr-update-error-properties)

File: src/app/components/error/index.html [hunk 1]
```diff
@@ -279,7 +279,7 @@ <h3>Guidelines</h3>
       defaultValue="false"
       [isOptional]="true"
     >
-      Indicates whether to replace the default title. If <sky-code>false</sky-code>, this title is appended to the default title. This property accepts <sky-code>boolean</sky-code> values.
+      Indicates whether to replace the default title. If <sky-code>false</sky-code>, this title is appended to the default title. To replace the default title, the property must be wrapped in brackets. This property accepts <sky-code>boolean</sky-code> values.
```
Comment (blackbaud-johnly) [inline src/app/components/error/index.html:282]: In the new sentence, I don't think we need the opening clause, but hopefully Alex can weigh in on what he thinks is the best way to communicate the this note about formatting in the code.

File: src/app/components/error/index.html [hunk 2]
```diff
@@ -289,7 +289,7 @@ <h3>Guidelines</h3>
       defaultValue="false"
       [isOptional]="true"
     >
-      Indicates whether to replace the default description. If <sky-code>false</sky-code>, this description is appended to the default description. This property accepts <sky-code>boolean</sky-code> values.
+      Indicates whether to replace the default description. If <sky-code>false</sky-code>, this description is appended to the default description. To replace the default description, the property must be wrapped in brackets. This property accepts <sky-code>boolean</sky-code> values.
```
Comment (blackbaud-johnly) [inline src/app/components/error/index.html:292]: Same thing as above about the opening clause in the new sentence.

File: src/app/components/error/index.html (hunk 1 above)
Comment (Blackbaud-DeniseCisneros) [reply src/app/components/error/index.html:282]: @Blackbaud-AlexKingman Would you mind sharing your preference/thoughts on this?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-update-error-properties](https://developer.blackbaud.com/skyux?_sp.spa=pr-update-error-properties)

File: src/app/components/error/index.html (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/components/error/index.html:282]: > To replace the default title, the property must be wrapped in brackets
I don't think this is necessary. Angular developers should know to use brackets for input properties. We don't call this out in other properties, so I don't think we'd need to do that here.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-update-error-properties](https://developer.blackbaud.com/skyux?_sp.spa=pr-update-error-properties)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-update-error-properties](https://developer.blackbaud.com/skyux?_sp.spa=pr-update-error-properties)

**Pull request merged**